src/dex-ui/pages/dao/CreateADAO/forms/CreateTokenDAOGovernanceForm.tsx

```diff
@@ -0,0 +1,326 @@
+              id: "governance.newToken.initialSupply",
+              label: "Initial token supply",
+              type: "number",
+              placeholder: "Enter amount",
+              unit: "$TOKEN",
+              isReadOnly: isFormInReadOnlyMode,
+              register: {
+                ...register("governance.newToken.initialSupply", {
+                  required: { value: true, message: "An initial token supply is required." },
+                }),
+              },
+            }}
+            formHelperText="If not specified, can be unlimited"
+            isInvalid={Boolean(errors.governance?.newToken?.initialSupply)}
+            errorMessage={
+              errors.governance?.newToken?.initialSupply && errors.governance?.newToken?.initialSupply.message
+            }
+          />
+          <Flex direction="row" gap="4" alignItems="flex-end">
+            <FormInput<"governance.newToken.supplyKey">
+              flex="5"
+              inputProps={{
+                flex: 5,
+                id: "governance.newToken.supplyKey",
+                label: "Supply Key",
+                type: "text",
+                isReadOnly: true,
+                placeholder: "Enter Supply Key",
+                register: {
+                  ...register("governance.newToken.supplyKey", {
+                    required: { value: true, message: "Supply key is required." },
+                  }),
+                },
+              }}
+              isInvalid={Boolean(errors.governance?.newToken?.supplyKey)}
+              errorMessage={errors.governance?.newToken?.supplyKey && errors.governance?.newToken?.supplyKey?.message}
+            />
+            <Button key="token-supply-key" onClick={handleGetAccountInfoClick} flex="1" marginBottom="8px">
+              Get Supply Key
+            </Button>
+          </Flex>
+        </DAOFormContainer>
+        <Text textStyle="p large regular">Initial token distribution</Text>
+        <DAOFormContainer>
+          <FormInput<"governance.newToken.treasuryWalletAccountId">
+            inputProps={{
+              id: "governance.newToken.treasuryWalletAccountId",
+              label: "Treasury wallet account id",
+              type: "text",
+              placeholder: "Enter wallet account id",
+              isReadOnly: isFormInReadOnlyMode,
+              register: {
+                ...register("governance.newToken.treasuryWalletAccountId", {
+                  required: { value: true, message: "A treasury account id is required." },
+                }),
+              },
+            }}
+            isInvalid={Boolean(errors.governance?.newToken?.treasuryWalletAccountId)}
+            errorMessage={
+              errors.governance?.newToken?.treasuryWalletAccountId &&
+              errors.governance?.newToken?.treasuryWalletAccountId.message
+            }
+          />
+        </DAOFormContainer>
+        <Flex gap="4" alignItems="center">
+          <Notification
```

Comment:
This warning notification text was intended to be temporary. You can remove this.

Reply:
Done, I too thought so but I was hesitant to make any last time changes in already approved UI so I  just simply edited  the text which matches the current context but now its removed.